dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/SourceCodeRepoInterface.java

```diff
@@ -332,7 +353,9 @@ public void updateEntryMetadata(final Entry<?, ?> entry, final DescriptorLanguag
             Workflow workflow = (Workflow)entry;
             workflow.getWorkflowVersions().forEach(workflowVersion -> {
                 String filePath = workflowVersion.getWorkflowPath();
-                updateVersionMetadata(filePath, workflowVersion, type, repositoryId);
+                if (!Objects.equals(SKIP_COMMIT_ID, workflowVersion.getCommitID())) {
```

Comment:
Totally optional, but only because it took me a bit to understand this: Either add a comment saying something to the effect that there is no need to change because the version has not changed, or create a function for the if condition whose name represents that.

I guess the disconnect is that cognitively SKIP_COMMIT_ID != unmodified to me.